### ISSUE TITLE
Problems with cache

### ISSUE THREAD

**Issue by Ruffeng** [shakacode/react_on_rails#658]
Hey @justin808 ,
I'm stuck on a stupid detail that i cannot figure out.
So far, when i was creating sth new or adding classes. Everything was working.
However, now I'm focused on css and it's not loading when i'm doing changes. It seems that is always reloading the same fingerprint css. I thought that i was sth about rails, until I realised that even on changes on my JSX files are cached now and i cannot see changes unless i write new lines there. So i'm uploading the same JS as well from fingerprints.

Do u know a way to disable the cache mode on webpack??? has anyone found this error as well?

Thanks in advance!

**Comment by justin808** [comment]
@Ruffeng Did you ever precompile your assets? 

http://stackoverflow.com/a/32107351/1009332

`rake assets:clobber`

**Comment by justin808** [comment]
@Ruffeng get in [touch with me](mailto:justin@shakacode.com) if you still have issues.

**Comment by Ruffeng** [comment]
Hi @justin808 . Sorry for my late reply. I wasn't able to check it proper. 
I used to deploy on heroku, so by default I have to precompile my assets to make it work ( I have to recognise that I'm using `precompile` and not` clobber`).
Finally I fixed migrating the project on a new one. I have no idea why all my project was like "frozen" in cache. I removed a lot of times the assets but it was still there.

Anyway, thanks Justin for your help :)

Ruffeng
